cf-oscore/src/test/java/org/eclipse/californium/oscore/ContextRederivationTest.java

```diff
@@ -79,6 +85,7 @@
 	private final static byte[] rid = new byte[] { 0x01 };
 
 	private static int SEGMENT_LENGTH = ContextRederivation.SEGMENT_LENGTH;
+	private static byte[] requestIdContext;
```

Comment:
would it be possible to move this into `RequestTestObserver`?

Reply:
Yes, I could do that. I made a new commit a09c1edbc4962b62c9d8be61e5da59f0c5e25de9
Does this fit with what you had in mind?

Reply:
I meant the field `requestIdContext`only, but keep the test (assert...) in the main thread.

Reply:
Okay, I understand. Like I have now updated it in commit 807413a38e3c224c999d33bc314bc6508d1ae89d?